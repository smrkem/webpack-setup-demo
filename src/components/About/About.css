.bio {
  line-height: 1.7;
  font-size: 1.8rem;
  color: #666;
}
