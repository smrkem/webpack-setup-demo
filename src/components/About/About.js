import React from 'react'
import './About.css'

const About = () => (
  <div className="container">
    <h1 className="heading">Hi karl, it's me. matt.</h1>
    <p className="bio">This section is all about me. yay!</p>
  </div>
)

export default About
