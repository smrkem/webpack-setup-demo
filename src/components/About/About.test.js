import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import About from './About'

test('About should be ok', () => {
  const component = shallow(<About />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
