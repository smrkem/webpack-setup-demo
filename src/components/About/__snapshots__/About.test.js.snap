// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should be ok 1`] = `
<div
  className="container"
>
  <h1
    className="heading"
  >
    Hi karl, it's me. matt.
  </h1>
  <p
    className="bio"
  >
    This section is all about me. yay!
  </p>
</div>
`;
